x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/40_unsupported_types.yml

```diff
@@ -175,25 +189,27 @@ unsupported:
   - match: { values.0.5: null }
   - match: { values.0.6: null }
   - match: { values.0.7: null }
-  - match: { values.0.8: null }
-  - match: { values.0.9: null }
+  - match: { values.0.8: "POINT (9.999999990686774 11.999999997206032)" }
```

Comment:
That's the problem with this approach, using doc values for reading points is loosy.

Reply:
Yes, and in fact my earlier experiment with getting from source started by fixing this test. We now have that planned as a followup PR.